modules/configuring-assumerole-acs.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="configuring-assumerole-acs_{context}"]
+= Configuring AssumeRole in {product-title-short}
+
+[role="_abstract"]
+After configuring AssumeRole in ECR, you can integrate {product-title} with Amazon Elastic Container Registry (ECR) by using AssumeRole.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Image Integrations* section, select *Amazon Elastic Container Registry*.
++
+The *Configure image integration* modal box opens.
+. Click *New Integration*.
+. Enter the details for the following fields:
+.. *Integration Name*: The name of the integration.
+.. *Types*: Select *Registry*.
+.. *Registry ID*: The ID of the registry.
+.. *Endpoint (Optional)*: The address of the registry.
+.. *Region*: The region for the registry.
+. Select the *Use AssumeRole* checkbox.
+. Depending wether the AssumeRole user in the same account as the ECR registry or in a different account, enter the details for the listed fields.
+.. same account as ECR:
+* *Account ID*: ECR ID.
+* *User*: The name of the user.
```

Comment:
All that needs to be supplied for AssumeRole is the role to be assumed and the registry where the role/ECR registry are (provided above). The account ID of the user is not needed because that information can be gained via either container IAM or from the access/secret key pair provided to auth as a user in AWS